packages/react-components/react-table/src/hooks/useTableColumnResizeMouseHandler.ts

```diff
@@ -2,6 +2,27 @@ import * as React from 'react';
 import { TableColumnId, ColumnResizeState } from './types';
 import { useFluent_unstable as useFluent } from '@fluentui/react-shared-contexts';
 
+type TouchOrMouseEvent = MouseEvent | TouchEvent;
+type ReactTouchOrMouseEvent = React.MouseEvent | React.TouchEvent;
+
+function isTouchEvent(event: TouchOrMouseEvent | ReactTouchOrMouseEvent): event is TouchEvent {
+  return ['touchstart', 'touchmove', 'touchend'].includes(event.type);
+}
+
+function isMouseEvent(event: TouchOrMouseEvent | ReactTouchOrMouseEvent): event is MouseEvent {
+  return ['mousedown', 'mousemove', 'mouseup'].includes(event.type);
+}
```

Comment:
honestly this could be inside `@fluentui/react-utilities`

Reply:
yup, I dont see why not. Will move it there